src/utils/localStorage.ts

```diff
@@ -0,0 +1,5 @@
+export function getLocalStorageItem(key: string) {
+  const item = window.localStorage.getItem(key);
+  const parsedItem = item && (JSON.parse(item) as string);
+  return parsedItem ?? '';
+}
```

Comment:
not necessary in this PR, but is it worth using `usePersistedState`?

https://github.com/gallery-so/gallery/blob/main/src/hooks/usePersistedState.ts

Reply:
I tried using persistedState at first, but for this use case I wanted a super simple way to just read a value from storage without syncing it to the state or anything, so I thought this util func might be useful